mysql.go

```diff
@@ -398,6 +398,18 @@ func (dialector Dialector) getSchemaIntAndUnitType(field *schema.Field) string {
 	return sqlType
 }
 
+func (dialector Dialector) getSchemaCustomType(field *schema.Field) string {
+	sqlType := string(field.DataType)
+
+	if !strings.Contains(strings.ToLower(sqlType), " auto_increment") {
```

Comment:
It would be better to check `AutoIncrement` first
```go
if field.AutoIncrement && !strings.Contains(strings.ToLower(sqlType), " auto_increment") {
```

Reply:
thank you